### PR TITLE
Minor: link to css-examples and github

### PR DESCRIPTION
**Pull request by estelle**
change from in-page code example to css-examples embedded example.
Changed link to see code to point to the new example code.

### CODE REVIEW

File: files/en-us/web/css/css_animations/index.md [hunk 1]
```diff
@@ -21,259 +21,9 @@ The animations CSS module lets you animate the values of CSS properties, such as
...
-  <i></i>
-  <i></i>
-  <i></i>
-  <i></i>
-  <i></i>
-  <div class="cloud"></div>
-  <div class="ground"><div>
-</div>
-```
+{{EmbedGHLiveSample("css-examples/modules/animation.html", '100%', 630)}}
```
Comment (wbamberg) [inline files/en-us/web/css/css_animations/index.md:24]: I see a scrollbar with this one, so I would give the iframe a bit more height

File: files/en-us/web/css/css_backgrounds_and_borders/index.md [hunk 2]
```diff
@@ -25,50 +25,9 @@ The properties in this module also let you define whether cells inside a {{HTMLE
...
+{{EmbedGHLiveSample("css-examples/modules/backgrounds.html", '100%', 430)}}
+
+The background images are defined with {{cssxref("background-image")}}. The images are centered with {{cssxref("background-position")}}. Different values of the {{cssxref("background-clip")}} property for the multiple background images are used to make the background images stay within the content box. The background color gets clipped to the padding box preventing the background from appearing through the transparent sections for the {{cssxref("border-image")}} and the {{cssxref("border-style", "dotted")}} {{cssxref("border")}}. The rounded corners in the element on the right are created using the {{cssxref("border-radius")}} property. A single {{cssxref("box-shadow")}} declaration is used to set all the shadows, both inset and outset. To see the code for this sample, [view the source on Github](https://github.com/mdn/css-examples/blob/main/modules/backgrounds.html).
```
Comment (wbamberg) [inline files/en-us/web/css/css_backgrounds_and_borders/index.md:30]: Consider starting a new paragraph for "To see the code..." ? For all of them really but this especially.

**Review by wbamberg — COMMENTED** [review]
Lovely examples. I think the iframe for the animations one needs to be a bit taller, and I had an optional text formatting suggestion.

File: files/en-us/web/css/css_animations/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/css/css_animations/index.md:24]:
```suggestion
{{EmbedGHLiveSample("css-examples/modules/animation.html", '100%', 650)}}
```

File: files/en-us/web/css/css_backgrounds_and_borders/index.md (hunk 2 above)
Comment (estelle) [inline files/en-us/web/css/css_backgrounds_and_borders/index.md:30]:
```suggestion
The background images are defined with {{cssxref("background-image")}}. The images are centered with {{cssxref("background-position")}}. Different values of the {{cssxref("background-clip")}} property for the multiple background images are used to make the background images stay within the content box. The background color gets clipped to the padding box preventing the background from appearing through the transparent sections for the {{cssxref("border-image")}} and the {{cssxref("border-style", "dotted")}} {{cssxref("border")}}. The rounded corners in the element on the right are created using the {{cssxref("border-radius")}} property. A single {{cssxref("box-shadow")}} declaration is used to set all the shadows, both inset and outset. 

To see the code for this sample, [view the source on Github](https://github.com/mdn/css-examples/blob/main/modules/backgrounds.html).
```

File: files/en-us/web/css/css_animations/index.md [hunk 3]
```diff
@@ -21,259 +21,9 @@ The animations CSS module lets you animate the values of CSS properties, such as
...
-}
-input:checked + label::before {
-  content: "Pause "
-}
-```
-
-{{ EmbedLiveSample('Animations_in_action', "630", "630") }}
-
-This sample animation uses {{cssxref("animation-iteration-count")}} to make the flakes fall repeatedly, {{cssxref("animation-direction")}} to make the cloud move back and forth, {{cssxref("animation-fill-mode")}} to raise the snow level in response to the cloud movement, and {{cssxref("animation-play-state")}} to pause the animation. To see the code for this animation, [view the source on Github](https://github.com/mdn/content/blob/main/files/en-us/web/css/css_animations/index.md?plain=1).
+This sample animation uses {{cssxref("animation-iteration-count")}} to make the flakes fall repeatedly, {{cssxref("animation-direction")}} to make the cloud move back and forth, {{cssxref("animation-fill-mode")}} to raise the snow level in response to the cloud movement, and {{cssxref("animation-play-state")}} to pause the animation. To see the code for this animation, [view the source on Github](https://github.com/mdn/css-examples/blob/main/modules/animation.html).
```
Comment (estelle) [inline files/en-us/web/css/css_animations/index.md:26]:
```suggestion
This sample animation uses {{cssxref("animation-iteration-count")}} to make the flakes fall repeatedly, {{cssxref("animation-direction")}} to make the cloud move back and forth, {{cssxref("animation-fill-mode")}} to raise the snow level in response to the cloud movement, and {{cssxref("animation-play-state")}} to pause the animation. 

To see the code for this animation, [view the source on Github](https://github.com/mdn/css-examples/blob/main/modules/animation.html).
```

File: files/en-us/web/css/css_backgrounds_and_borders/index.md [hunk 4]
```diff
@@ -25,50 +25,11 @@ The properties in this module also let you define whether cells inside a {{HTMLE
...
+{{EmbedGHLiveSample("css-examples/modules/backgrounds.html", '100%', 430)}}
+
+The background images are defined with {{cssxref("background-image")}}. The images are centered with {{cssxref("background-position")}}. Different values of the {{cssxref("background-clip")}} property for the multiple background images are used to make the background images stay within the content box. The background color gets clipped to the padding box preventing the background from appearing through the transparent sections for the {{cssxref("border-image")}} and the {{cssxref("border-style", "dotted")}} {{cssxref("border")}}. The rounded corners in the element on the right are created using the {{cssxref("border-radius")}} property. A single {{cssxref("box-shadow")}} declaration is used to set all the shadows, both inset and outset.
```
Comment (wbamberg) [inline files/en-us/web/css/css_backgrounds_and_borders/index.md:30]:
```suggestion
The background images are defined with {{cssxref("background-image")}}. The images are centered with {{cssxref("background-position")}}. Different values of the {{cssxref("background-clip")}} property for the multiple background images are used to make the background images stay within the content box. The background color gets clipped to the padding box preventing the background from appearing through the transparent sections for the {{cssxref("border-image")}} and the {{cssxref("border-style", "dotted")}} {{cssxref("border")}}. The rounded corners in the element on the right are created using the {{cssxref("border-radius")}} property. A single {{cssxref("box-shadow")}} declaration is used to set all the shadows, both inset and outset.
```

**Review by wbamberg — APPROVED** [review]
👍

**Pull request merged**